recipes/keycloack-gatekeeper/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set goname = "github.com/keycloak/keycloak-gatekeeper" %}
+{% set version = "2.3.0" %}
+
+{% set name = goname.split('/')[-1] %}
+{% set pkg_src = ('src/'+goname).replace("/", "\\" if win else "/") %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - folder: {{ pkg_src }}
+    url: https://{{ goname}}/archive/v{{ version }}.tar.gz   # [unix]
+    sha256: cb19406a35797165fb135a5152c8e6ace17315f892ad5a3d0e13b89ca9aae2cd    # [unix]
+    git_url: https://{{ goname }}  # [win]
+    git_tag: v{{ version }}  # [win]
+
+build:
+  number: 0
+  script:
+    - pushd {{ pkg_src }}
+    - go install -v .
```

(1/2) According to upstream [Makefile](https://github.com/keycloak/keycloak-gatekeeper/blob/d8a5f554db5ac134368a5c1e2bb5c24501a207c0/Makefile#L83-L85) we need to to call `dep ensure` before actually installing the package. 
```suggestion
    - dep ensure
    - go install -v .
```